app/models/article.rb

```diff
@@ -112,7 +112,7 @@ class Article < ApplicationRecord
 
   scope :admin_published_with, lambda { |tag_name|
     published
-      .where(user_id: SiteConfig.staff_user_id)
+      .where(user_id: User.with_role(:super_admin).union(User.with_role(:admin)).select(:id))
```

Comment:
Is the mascot guaranteed to be an admin? Want to make sure this won't break any Forems where the mascot might not have admin privileges. @pkfrank you might be able to answer this as well.

Reply:
@mstruve the Mascot does not appear to have admin privileges out of the gate (checking via `/admin/users` on a fresh Forem from yesterday).  Perhaps @vaidehijoshi or @atsmith813 can chime in on whether we'd want to add that to the Mascot?

Reply:
If we want to add it, it can go here:

https://github.com/forem/forem/blob/4dc1a5375519221f0bf60e210e79762d580f292f/app/services/users/create_mascot_account.rb#L17-L22

with `mascot.add_role(:super_admin)`

Reply:
Whew, thanks for the insight @atsmith813. Rather than add it there and make a DataUpdateScript for all forems, I will update the code to check for that ID as well which actually should make the test changes then unnecessary.